README.md

```diff
@@ -24,78 +27,91 @@ A Simple Terminal-Based Mod Organizer for Linux
 
-```
-activate   mod|plugin <index>             Enabled components will be loaded by game.
-commit                                    Apply pending changes.
-configure  <index>                        Configure a fomod.
-deactivate mod|plugin <index>             Disabled components will not be loaded by game.
-delete     mod|download <index>           Removes specified file from the filesystem.
-exit                                      Quit.
-find       [<keyword> ... ]               Fuzzy filter. 'find' without args removes filter.
-help                                      Show this menu.
-install    <index>                        Extract and manage an archive from ~/Downloads.
-move       mod|plugin <index> <new_index> Larger numbers win file conflicts.
-refresh                                   Abandon pending changes.
-rename     mod|download <index> <name>    Names may contain alphanumerics and underscores.
-```
-
-# Tips and Tricks
+| Command     | Arguments                           | Description |
+|-|-|-|
+| activate    | mod/plugin `<index>`                | Enabled components will be loaded by game |
+| commit      |                                     | Apply pending changes |
+| configure   | `<index>`                           | Configure a fomod |
+| deactivate  | mod/plugin `<index>`                | Disabled components will not be loaded by game |
+| delete      | mod|download `<index>`              | Removes specified file from the filesystem |
+| exit        |                                     | Quit. Prompts if there are changes |
+| find        | [`<keyword>` ...]                   | Show only components with any keyword. `find` without args resets.
+| help        |                                     | Show this menu |
+| install     | `<index>`                           | Extract and manage an archive from ~/Downloads |
+| move        | mod|plugin [from_index] [to_index]  | Larger numbers win file conflicts |
+| refresh     |                                     | Abandon pending changes |
+| vanilla     |                                     | Disable all managed components and clean up |
+
+Note that the `de/activate mod|plugin` command now supports `all` in place of `<index>`.
+This will activate or deactivate all mods or plugins that are visible. Combine this
+with the `find` command to quickly organize groups of components with related names.
+
+### Usage Tips and Tricks
 
 - Note that the `de/activate mod|plugin` command supports `all` in place of `<index>`.
   This will activate or deactivate all mods or plugins that are visible. Combine this
   with the `find` command to quickly organize groups of components with related names.
   You can leverage this to automatically sort your plugins to the same order as your
   mod list:
-  ```
-  deactivate mod all
-  # sort your mods with the move command
-  activate mod all
-  activate plugin all
-  commit
-  ```
+
+    ```bash
```

Comment:
We're using `sh` above, but using `bash` here. Is there a reason for that? If not, we should consolidate around one or the other. Don't care which :)

Reply:
Ah no the bash was just because thats what my muscle memory defaults to when typing haha I'll unify it with the other `sh` instances

Reply:
Commit 34ac985 changes all block lables to be `sh`